services/uploads/package.json

```diff
@@ -12,8 +12,12 @@
         "serverless-iam-helper": "CMSgov/serverless-iam-helper",
         "serverless-idempotency-helper": "CMSgov/serverless-idempotency-helper",
         "serverless-plugin-scripts": "^1.0.2",
-        "serverless-s3-local": "^0.6.7",
         "serverless-s3-bucket-helper": "CMSgov/serverless-s3-bucket-helper",
+        "serverless-s3-local": "^0.6.7",
         "serverless-stack-termination-protection": "^1.0.4"
+    },
+    "dependencies": {
+        "copy-webpack-plugin": "^10.2.4",
+        "serverless-webpack": "^5.7.0"
```

Comment:
Would like to be able to remove this?  Added these temporarily to resolve module not found errors coming up in uploads service.  This issue was keeping the application from building. However, these deps I think were originally added for prisma in app-api. Not clear why they are needed here.